### ISSUE TITLE
Hey, can the ''batch_norm'' you defined work the same as the official nn.batchnorm2d?

### ISSUE THREAD

**Issue by snowaner** [0zgur0/conditional_batch_normalization#4]
Like saving the mean and variance of the whole training set.
